### ISSUE TITLE
uninitialized constant Chikka::Response::JSON (NameError)

### ISSUE THREAD

**Issue by eebasadre20** [scrambledeggs/chikka#2]
Hi, I tried to use this chikka gem for my project. But when I try to run the file I got this error "`initialize': uninitialized constant Chikka::Response::JSON (NameError)".

**Comment by dreinavarro** [comment]
Can you show how you are using the gem?

**Comment by eebasadre20** [comment]
I just followed your documentation. `gem install chikka` .. However, I tried to copy your code to my own file and I found out that it requires `require 'json'` for the error to be resolve, but yet I got another error `BadRequestError : Invalid Test Mobile Number` .. I don't know but I already followed the format for mobile_number.

`chikka-msg.rb:78:in`parse': Invalid Test Mobile Number (BadRequestError)
from chikka-msg.rb:61:in `send_message'
from chikka-msg.rb:87:in`<main>'`

**Comment by x44x45x41x4E** [comment]
@eebasadre20 Care to gist it?

**Comment by adeleneleda** [comment]
@eebasadre20 I encountered the same error: "BadRequestError : Invalid Test Mobile Number" when trying to integrate this with a sample app, it seems that it is already from the API. I think we need to first set a test mobile number in our Chikka accounts before we can send messages. :) After I set set mine, I already was able to send messages. :)

Hope this helps!

Set your number in your account settings (similar to the following):
<img width="967" alt="screen shot 2016-05-04 at 11 16 30 pm" src="https://cloud.githubusercontent.com/assets/1901859/15018907/4a3ac45a-124e-11e6-86bc-2c6495ab7f02.png">
